src/main/java/ch/epfl/biop/imaris/EasyXT.java

```diff
@@ -774,6 +782,85 @@ public static IDataSetPrx getSurfacesDataset(ISurfacesPrx surface, double downsa
         return data;
     }
 
+    /**
+     * Get an ImagePlus of the spots as a mask (255)
+     *
+     * @param spots , a spots object see {@link  #getSpots(String)}
+     * @return , an ImagePlus
+     * @throws Error
+     */
+    public static ImagePlus getSpotsMask(ISpotsPrx spots) throws Error {
+        return getSpotsImage(spots, false, false);
+    }
+
+    // TODO Comment
+    public static ImagePlus getSpotsLabel(ISpotsPrx spots) throws Error {
+        return getSpotsImage(spots, true, false);
+    }
+
+    // TODO Comment
+    public static ImagePlus getSpotsImage(ISpotsPrx spots, boolean isValueID, boolean isGauss) throws Error {
+        // Prepare the imp to receive spots pixels
+        //
+        // get the calibration info from Imaris
+        ImarisCalibration cal = new ImarisCalibration(app.GetDataSet());
+        // Create a new imp with the cal and get stack ()
+        //ImagePlus imp = IJ.createHyperStack( getOpenImageName(), cal.xSize , cal.ySize, cal.cSize, cal.zSize, cal.tSize , 32);
+        //imp.setCalibration( cal );
+        //ImageStack stack = imp.getStack();
+
+        long[] spots_ids = spots.GetIds();
+        float[][] spots_centerXYZ = spots.GetPositionsXYZ();
+        float[][] spots_radiiXYZ = spots.GetRadiiXYZ();
+        int[] spots_t = spots.GetIndicesT();
+
+        double resXY = cal.pixelWidth;
+        double resZ = cal.pixelDepth;
+        String unit = cal.getUnit();
+
+        // Define default vectors
```

Comment:
Perhaps a more explicit explanation of these vectors would be nice. 
`Define axes for the spots. They can be arbitrary, but in this case, we make two 'trivial' unit vectors defining the X and Y axes.